test/unit/dialects/mssql/query-generator.test.js

```diff
@@ -159,7 +163,7 @@ if (current.dialect.name === 'mssql') {
       });
 
       // Test older version (< v11.0.0)
-      modifiedGen.sequelize.options.databaseVersion = '10.0.0';
+      dbVersion = '10.0.0';
```

Comment:
This doesn't do what you hope it does: `modifiedGen.sequelize.options.databaseVersion` remains unchanged when it's the value that will be used by sequelize. Updating `dbVersion` instead does nothing because that variable is never used again